Mage/src/main/java/mage/game/draft/DraftImpl.java

```diff
@@ -263,7 +306,7 @@ public void addPlayerQueryEventListener(Listener<PlayerQueryEvent> listener) {
     public void firePickCardEvent(UUID playerId) {
         DraftPlayer player = players.get(playerId);
         int cardNum = Math.min(15, this.cardNum);
-        int time = timing.getPickTimeout(cardNum);
+        int time = timing.getPickTimeout(cardNum) - boosterLoadingCounter * BOOSTER_LOADING_INTERVAL;
```

Comment:
Why is the number of attempts being subtracted from the amount of time they have available?

Reply:
In case the first booster loading doesn't get a reply from the player's client, it's sent again. The pick timer is substracted accordingly, to not show a time to the player that's too long.